shared/chat/conversation/container.js

```diff
@@ -27,7 +27,7 @@ class Conversation extends React.PureComponent<SwitchProps> {
     }
     switch (this.props.type) {
       case 'error':
-        return this.props.conversationIDKey && <Error conversationIDKey={this.props.conversationIDKey} />
```

Comment:
conversationIDKey is isn't nullable, it should never be '', and Error handles '' fine anyway.